### PR TITLE
Add more options to scripts

### PR DESCRIPTION
**Pull request by sbquinlan**
This change adds support for more options in scripts:
- ```poetry run list-ingest```: Lists the currently running ingestions
- ```poetry run get-ingest <id>```: Gets the status and message of a particular ingestion
- ```poetry run delete-ingest <id>```: Stops an ingestion if it is still running
- ```poetry run delete-collection <query>```: Deletes collections from the stac catalog

Verified by running all the scripts against the test tier.

### CODE REVIEW

File: scripts/ingest.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    ingestor = IngestionApi.from_veda_auth_secret(
+        secret_id=os.environ.get("COGNITO_APP_SECRET"),
+        base_url=os.environ.get("STAC_INGESTOR_API_URL"),
+    )
+    response = ingestor.request(
+        "GET",
+        "/ingestions",
+    )
+    response.raise_for_status()
+    print(json.dumps(response.json(), indent=2))
```
Comment (jjfrench) [inline scripts/ingest.py:21]: Did you also want a `try`:`except` around this?

**Comment by jjfrench** [comment]
Looks like there are also a couple of added collections, did we want to add those to another PR or include them along with this PR scope?

**Comment by sbquinlan** [comment]
@jjfrench yeah the added collections are just from poorly rebasing onto add-upstream-changes, i'll rebase those out.

**Review by jjfrench — APPROVED** [review]
Optionally add more type hints to collection.py/ingest.py, but looks good to me

**Review by emileten — APPROVED** [review]
This is cool, thanks !

**Comment by emileten** [comment]
@sbquinlan it's a bit weird, this appears to have the add-upstream changes that were merged to main, though 🤔 wondering if it will be fixed by rebasing onto main ?

**Comment by sbquinlan** [comment]
Yeah I'll rebase manually before landing. Github isn't smart enough to autosquash stacked PRs :(

**Pull request merged**